examples/protos/narwhal.proto

```diff
@@ -58,7 +58,7 @@ message ValidatorData {
 
 message CollectionRetrievalResult {
     oneof retrieval_result {
-        BatchMessage batch = 1;
+        BatchMessage transaction_list = 1;
```

Comment:
The problem here is not so much the name 'batch', it's the reuse of "BatchMessage". I.e. this still makes folks believe we are reusing the Batch concept to convey this transaction list. I think it's better to use a different type.